packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -94,17 +94,21 @@ export class CdkToolkit {
         throw new Error(`There is no file at ${options.templatePath}`);
       }
       const template = deserializeStructure(await fs.readFile(options.templatePath, { encoding: 'UTF-8' }));
-      diffs = printStackDiff(template, stacks.firstStack, strict, contextLines, stream);
+      diffs = options.securityCheckOnly
+        ? +printSecurityDiff(template, stacks.firstStack, RequireApproval.Broadening)
```

Comment:
+? :)

Reply:
Returns true, so I use the + operator to typecast to a number